api/src/main/resources/messages.properties

```diff
@@ -46,13 +45,40 @@ adminui.addGlobalProperty.label = Add Global Property
 adminui.editGlobalProperty.label =Edit Global Property
 
 adminui.rolesAndPrivileges.title=Roles And Privileges
+
+adminui.managePrivileges.title = Manage Privileges
+
+adminui.manageRoles.title = Manage Roles
+adminui.addNewRole.label=Add New Role
+adminui.editRole.label=Edit Role
+adminui.role.addNewRole.title=Add New Role
+adminui.role.editRole.title=Edit Role
+adminui.role.retireRole.title=Retire Role
+adminui.role.purge.title=Delete Role
+adminui.role.role = Role
+adminui.role.description = Description
+adminui.role.inheritedRoles = Inherited Roles
+adminui.role.privileges = Privileges
+adminui.role.containedRoles.label = Roles that contain (inherit privileges from)
```

Comment:
This is how the old ui works. In the old ui, there are 4 sections which is replicated in the new ui:
base roles: role which inherit from the current role
inherited roles: roles which the current role inherits from
inherited privileges: privileges which the current role inherits and which cannot be changed directly
privileges: privileges attached to current role, which can be edited